Sources/ArgumentParser/Parsable Properties/OptionGroup.swift

```diff
@@ -31,6 +31,7 @@
 @propertyWrapper
 public struct OptionGroup<Value: ParsableArguments>: Decodable, ParsedWrapper {
   internal var _parsedValue: Parsed<Value>
+  internal var _hidden: Bool?
```

Comment:
Let's have this be non-optional, and just default to `false`. Can you call it something like `_hiddenFromHelp`?